### ISSUE TITLE
SearchableSnapshotsBlobStoreCacheIntegTests.testBlobStoreCache failure

### ISSUE THREAD

**Issue by javanna** [elastic/elasticsearch#70014]
**Build scan**:

https://gradle-enterprise.elastic.co/s/copza7hki4jye/tests/:x-pack:plugin:searchable-snapshots:internalClusterTest/org.elasticsearch.blobstore.cache.SearchableSnapshotsBlobStoreCacheIntegTests/testBlobStoreCache#1

**Repro line**:

./gradlew ':x-pack:plugin:searchable-snapshots:internalClusterTest' --tests "org.elasticsearch.blobstore.cache.SearchableSnapshotsBlobStoreCacheIntegTests.testBlobStoreCache" -Dtests.seed=F822E829D044CF2 -Dtests.security.manager=true -Dtests.locale=fr-CH -Dtests.timezone=Europe/Nicosia -Druntime.java=8

**Reproduces locally?**:

Yes

**Applicable branches**:

7.x

**Failure excerpt**:

```

java.lang.AssertionError: {"file_ext":"liv","num_files":1,"total_size":1147,"open_count":1,"close_count":1,"contiguous_bytes_read":{"count":2,"sum":1147,"min":123,"max":1024},"non_contiguous_bytes_read":{"count":0,"sum":0,"min":0,"max":0},"cached_bytes_read":{"count":1,"sum":123,"min":123,"max":123},"index_cache_bytes_read":{"count":1,"sum":1024,"min":1024,"max":1024},"cached_bytes_written":{"count":2,"sum":1147,"min":123,"max":1024,"time_in_nanos":33606204},"direct_bytes_read":{"count":0,"sum":0,"min":0,"max":0,"time_in_nanos":0},"optimized_bytes_read":{"count":0,"sum":0,"min":0,"max":0,"time_in_nanos":0},"forward_seeks":{"small":{"count":0,"sum":0,"min":0,"max":0},"large":{"count":0,"sum":0,"min":0,"max":0}},"backward_seeks":{"small":{"count":0,"sum":0,"min":0,"max":0},"large":{"count":0,"sum":0,"min":0,"max":0}},"blob_store_bytes_requested":{"count":1,"sum":123,"min":123,"max":123},"current_index_cache_fills":0} |

Expected: <0L>
but: was <1L>


```

**Comment by ywelsch** [comment]
Thanks @javanna. No need to mute, will fix that right away

**Comment by ywelsch** [comment]
fixed by #70017
